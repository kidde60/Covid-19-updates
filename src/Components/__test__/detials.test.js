// import renderer from 'react-test-renderer';
// import { Provider } from 'react-redux';
// import '@testing-library/jest-dom';
// import Details from '../../Components/Details/Details';
// import store from './mock/configureStore';

// it('Check if the component has changed', async () => {
//   const tree = renderer.create(
//     <Provider store={store}>
//       <Details />
//     </Provider>,
//   ).toJSON();
//   expect(tree).toMatchSnapshot();
// });
