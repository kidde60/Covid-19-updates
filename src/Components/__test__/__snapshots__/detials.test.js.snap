// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Country details Check if it renders correctly 1`] = `
<div>
  <div
    className="background"
  >
    <div>
      <div
        className="Details"
      >
        <div
          className="flex"
        >
          <h2>
            Country
          </h2>
          <h2>
            Afghanistan
          </h2>
        </div>
        <div
          className="flex"
        >
          <h2>
            Last Updated
          </h2>
          <h2
            className="dt"
          >
            2022-10-21T18:09:47.252Z
          </h2>
        </div>
        <div
          className="flex"
        >
          <h2>
            New Confirmed Cases
          </h2>
          <h2>
            193
          </h2>
        </div>
        <div
          className="flex"
        >
          <h2>
            New Deaths
          </h2>
          <h2>
            2
          </h2>
        </div>
        <div
          className="flex"
        >
          <h2>
            New Recovery
          </h2>
          <h2>
            0
          </h2>
        </div>
        <div
          className="flex"
        >
          <h2>
            Total Confirmed Cases
          </h2>
          <h2>
            201750
          </h2>
        </div>
        <div
          className="flex"
        >
          <h2>
            Total Deaths
          </h2>
          <h2>
            7814
          </h2>
        </div>
      </div>
    </div>
  </div>
</div>
`;
